assignments/closure.js

```diff
@@ -1,5 +1,13 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const fn = "Sachin";
+function fullname()
+{
+  const ln = "Benny";
+  console.log(`${fn} ${ln}`);
+}
+fullname();
```

Comment:
Remember, you don't need to have a variable in the global scope to have a closure. Something like this would work too:

```js
function fullName() {
    const fn = "Sachin";

    return function sayName() {
        const ln = "Benny";
        return `${fn} ${ln}`;
    }
}

const sayName = fullName();
console.log(sayName()); // Sachin Benny
```